types/config.go

```diff
@@ -87,6 +87,7 @@ type Config struct {
 		Endpoint     string           `yaml:"endpoint" envconfig:"BEACONAPI_ENDPOINT"`
 		Endpoints    []EndpointConfig `yaml:"endpoints"`
 		EndpointsURL string           `yaml:"endpointsUrl" envconfig:"BEACONAPI_ENDPOINTS_URL"`
+		ClientIndex  int              `yaml:"clientIndex" envconfig:"BEACONAPI_CLIENT_INDEX"`
```

Comment:
Hack that we use to fix the node we want to always connect to.

Note: The alternative here would be to not put Dora in the kurtosis config and just running it separately connecting to the specific node we care about.